src/test/java/winstone/accesslog/SimpleAccessLoggerTest.java

```diff
@@ -40,8 +40,14 @@ public void testSimpleConnection() throws Exception {
         makeRequest("http://localhost:"+port+"/examples/CountRequestsServlet");
 
         // check the log file
-        Thread.sleep(5000);
-        String text = FileUtils.readFileToString(logFile, StandardCharsets.UTF_8);
+        // check the log file every 100ms for 5s
+        String text = "";
+        for(int i=0; i < 50; ++i) {
```

Comment:
the idea of a loop should be to wait a certain time until reaching a conditon (there is no timeout with such loop)
if you don't want to do it manually this can be easily achieved using http://www.awaitility.org/